packages/bedrock-components/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/bedrock-components",
   "title": "Bedrock Components",
-  "version": "1.3.7",
+  "version": "1.3.7-16",
```

```suggestion
  "version": "1.3.8",
```
newest version needs to be 1.3.8